crosswalks/Citation_File_Format_1.2.0.README.md

```diff
@@ -0,0 +1,183 @@
+CodeMeta is ambiguous for this term (see <https://github.com/codemeta/codemeta/issues/264>), and it is unclear which value of the terms should be mapped if they differ.
+
+#### CFF to CodeMeta
+
+CFF `version` maps to both CodeMeta `version` and CodeMeta `software Version`.
+CodeMeta is ambiguous for this term (see <https://github.com/codemeta/codemeta/issues/264>).
+Until this is resolved, CFF `version` should probably be mapped to both.
+
+### `isPartOf`/`hasPart` <=> `identifiers`
+
+CFF doesn't (yet, see  https://github.com/citation-file-format/citation-file-format/issues/69#issuecomment-904546233) support these qualifiers for `identifiers` in a machine-readable way.
+`identifiers` is a list of identifiers pertaining to the software ("The identifiers of the software or dataset.").
+
+As of version 1.2.0, CFF `identifiers` may have an `identifier.description`, which may specify the relation of the identifier to another object,
+and which may be mapped to either `isPartOf` or `hasPart`.
+
+#### CodeMeta to CFF
+
+If the `schema:CreativeWork` for CodeMeta `isPartOf` or `hasPart` contain an identifier or definitive URL, 
+these maps to a CFF `identifier`.
+The target type of the CFF `identifier` could be heuristically determined based on the identifier in the CodeMeta value.
+The qualifying part `hasPart` and `isPartOf` can be encoded (automatically) in CFF `identifier.description` in plain text.
+
+#### CFF to CodeMeta
+
+Values in CFF `identifiers` map to either CodeMeta `hasPart` or CodeMeta `isPartOf`, 
+if the respective relationship can be determined from the CFF `identifier.description`, either manually or heuristically.
+
+### `identifier` <=> `identifiers`/`person.orcid`/`doi`
+
+Mapping depends on how a CodeMeta instance uses the term, which is ambiguously documented ("URL identifer, ideally an ORCID ID for individuals, a FundRef ID for funders").
+
+#### CodeMeta to CFF
+
+If the CodeMeta `identifier` is an identifier for the described software, it maps to CFF `identifiers`.
+If the CodeMeta `identifier` is a DOI for the described software, it maps to either CFF `doi` or a CFF `identifier` of CFF `identifier.type` `doi`.
+If the CodeMeta `identifier` is the identifier for a person (e.g., an author), and is an ORCID id, it maps to CFF `person.orcid`.
+
+#### CFF to CodeMeta
+
+CFF `person.orcid` maps to CodeMeta `identifier` of the respective person.
+A single CFF `identifier` maps to `identifier`.
+The respective CFF `identifier` to use for the mapping needs to be determined manually or heuristically from the list of available CFF `identifiers`, 
+e.g., if there is only one identifier, and no value for CFF `doi` is supplied, or if the single identifier is of `identifier.type` `doi` 
+and has the same value as `doi` (but `doi` may override this behaviour, see next sentence).
+A CFF `doi` maps to `identifier` and should be used even in the case where there additionally are CFF `identifiers`.
```

Comment:
not sure I agree with the prioritization there, as CFF's `doi` is inherently more ambiguous than CFF's `identifiers[i].type==doi`, since the latter has a `description` (and perhaps a machine readable `qualifier` / `role` in the future)

Reply:
I half-agree. You're right about the potentially greater disambiguation with `identifiers`, on the other hand, the principles clearly prefer `doi` (although in practice they may be more ambiguous). I've solved this by removing prioritization wording altogether, to leave this to the user. (0cb24150d3a36b70a499b5e4db1fff09738c30b2)